### DISCUSSION TITLE
Query Manager does not show history between sessions?

### DISCUSSION THREAD

**Discussion by tatzio** [dbeaver/dbeaver#17570] [Q&A] [answered]
Hi, I have enabled saving query log into the file:

![image](https://user-images.githubusercontent.com/106803609/188575641-31136629-3136-405f-991c-c6bddd582a35.png)

All queries are logged, even those from previous sessions.
But when I restart DB, only new queries from current session are shown on the Query Managers list.
Is there an option to enable showing/searching queries from all previous sessions?

**Comment by TheMechanist902** [comment, marked as answer]
Hi,
I think this is a feature exclusive to the [Enterprise Edition](https://dbeaver.com/docs/wiki/Enterprise-Edition/):

> Additional EE plugins:
> 
> - [...]
> - **Persistent query database. Search in query history**
> - [...]

Maybe the log file can be used manually. I never tried this yet.
